.github/workflows/pr.yml

```diff
@@ -44,6 +44,8 @@ jobs:
         run: yarn -s ts-circular-deps:check
       - name: Check code format
         run: yarn ng-dev format changed --check ${{ github.event.pull_request.base.sha }}
+      - name: Check Package Licenses
```

Comment:
Nit: it's typically best to have at the end, in case we need to skip it for the reasons you saw today.

Reply:
It's _almost_ at the end already. The commit message check is intentionally last, because the caretaker can override it.

Reply:
Makes sense